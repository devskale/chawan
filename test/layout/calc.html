<!DOCTYPE html>
<div style="width: 20ch">
<!-- valid -->
<div style="width: calc(0em - 10em); background: red">0ch</div>
<div style="width: calc(10ch - 3em); background: red">4ch</div>
<div style="width: calc(1ch * 2); background: red">2ch</div>
<div style="width: calc(1ch + calc(2ch - 1ch)); background: red">2ch</div>
<div style="width: calc(1ch * 2 + calc(2ch - 1ch)); background: red">3ch</div>
<!-- invalid -->
<div style="width: calc(1 * 2); background: red">test</div>
<div style="width: calc(); background: red">test</div>
<div style="width: calc(1); background: red">test</div>
<div style="width: calc(1ch + + 1ch); background: red">test</div>
