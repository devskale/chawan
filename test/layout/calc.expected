0ch
[48;2;255;0;0m4ch [49m
[48;2;255;0;0m2c[49mh
[48;2;255;0;0m2c[49mh
[48;2;255;0;0m3ch[49m
[48;2;255;0;0m8ch     [49m
[48;2;255;0;0m1[49mch
[48;2;255;0;0m2c[49mh
[48;2;255;0;0m1[49mch
[38;2;255;40;40mred[39m
[38;2;255;40;40mred[39m
[38;2;255;40;40mred[39m
[38;2;255;40;40mred[39m
[48;2;255;0;0mtest                [49m
[48;2;255;0;0mtest                [49m
[48;2;255;0;0mtest                [49m
[48;2;255;0;0mtest                [49m
[48;2;255;0;0mtest                [49m
test
red
